### PR TITLE
feat: [IDP]: Added the info on how to add annotations

### PR DESCRIPTION
**Pull request by Debanitrkl**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md [hunk 1]
```diff
@@ -669,6 +669,26 @@ POST /catalog/custom-properties/entity
...
+{
+  "entity_ref": "github-pull-request",
+  "property": "metadata.annotations",
+  "value": {
+          "harness.io/pipelines": "CI: https://app.harness.io/ng/account/account_id/module/idp-admin/orgs/default/projects/idpprojectsc/pipelines/DummyPipeline/pipeline-studio?storeType=INLINE"
+        },
+  "mode": "append"
+}
+```
+Result: The `metadata.annotations` property will now add `harness.io/pipelines` to the existing annotations.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md:690]: What happens if the annotation already exists?

File: docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md [hunk 2]
```diff
@@ -669,6 +669,26 @@ POST /catalog/custom-properties/entity
...
+```http
+POST /catalog/custom-properties/entity
+```
+
+```json title="Payload" {4}
+{
+  "entity_ref": "github-pull-request",
+  "property": "metadata.annotations",
+  "value": {
+          "harness.io/pipelines": "CI: https://app.harness.io/ng/account/account_id/module/idp-admin/orgs/default/projects/idpprojectsc/pipelines/DummyPipeline/pipeline-studio?storeType=INLINE"
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md:685]: Can we show an example of 2 or more pipelines?

File: docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md (hunk 1 above)
Comment (Debanitrkl) [reply docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md:690]: It overwrites the same

File: docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md (hunk 2 above)
Comment (Debanitrkl) [reply docs/internal-developer-portal/catalog/catalog-ingestion/catalog-ingestion-api.md:685]: That seems to be not working at the moment we are checking will add it here once it's fixed.

**Pull request merged**